src/classes/LD_LeadConvertOverride_CTRL.cls

```diff
@@ -179,17 +184,48 @@ public with sharing class LD_LeadConvertOverride_CTRL {
     private map<id, contact> conMap {
         get {
             if (conMap == null) {
+                Set<String> fieldSet = new Set<String>{'Id', 'Name', 'FirstName', 'LastName'};
+                String queryStr = 'SELECT ';
+                for (FieldSetMember cfs : conFieldSet)
+                   fieldSet.add(cfs.getFieldPath());
+                queryStr += String.join(new List<String>(fieldSet), ',');
+                queryStr += ' FROM Contact WHERE (LastName = \'' + l.LastName + '\' AND FirstName = \'' + l.FirstName + '\') OR (Email = \'' + l.Email + '\' AND Email != null) LIMIT 50';
```

Comment:
sorry for not catching this last review.  you should avoid concatenating fields directly in soql strings that could potentially be malicious or ill formatted.  your dynamic soql should still use variable reference syntax.  Ie, '...where Lastname = :l.llastname...'